autoharness/orchestrator/test_orchestrator.py

```
@@ -0,0 +1,374 @@
+
+BASE = "http://localhost:3111"
+TAG = f"test-{int(time.time())}"
+PASS = 0
+FAIL = 0
+
+
+def api(path, data=None, method="POST"):
+    url = f"{BASE}{path}"
+    body = json.dumps(data).encode() if data else None
+    headers = {"Content-Type": "application/json"} if body else {}
+    req = urllib.request.Request(url, data=body, headers=headers, method=method)
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)